examples/db_merge.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Let's name this one merge_01.json, other merge_02.json
Add a description at the top of the file

"description" : " This graph has this and that"